### PR TITLE
Update certificate-connector-install.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the sentence for the intune license requirement as it creates ambiguity.

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4239

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/protect/certificate-connector-install.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Use the following procedure to both configure a new connector and modify a previ
 
 5. On the *Prerequisites* page, the wizard runs several checks on the server before the configuration can begin. Review and resolve any errors or warnings before you continue.
 
-6. On the *Azure AD Sign In* page, select the environment that hosts your Azure Active Directory, and then select **Sign In**. You’ll then be asked to authenticate your access. This user account must be a Global Admin or an Intune Admin with an Intune license assigned.
+6. On the *Azure AD Sign In* page, select the environment that hosts your Azure Active Directory, and then select **Sign In**. You’ll then be asked to authenticate your access. An Intune license is required for the account that you are signing in with that can be either a Global Administrator or Intune Administrator.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/certificate-connector-install.md:115]:
```suggestion
6. On the *Azure AD Sign In* page, select the environment that hosts your Azure Active Directory, and then select **Sign In**. You’ll then be asked to authenticate your access. An Intune license is required for the account that you sign in with which can be either a Global Administrator or an Intune Administrator.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and signoff this PR.  Thanks

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**